build.gradle

```diff
@@ -41,8 +41,13 @@ dependencies {
 }
 
 javafx {
-    version = "11.0.1"
-    modules = [ 'javafx.controls' ]
+    version = "11.0.2"
```

Comment:
Shouldn't it be 11.0.6 to avoid that HTTP problem?

Reply:
@colinanderson this is the JavaFX version, not the JDK/JRE version

11.0.2 is LTS version of JavaFX

Reply:
@colinanderson actually I'm wrong, not LTS, but public version is 11.0.2.

https://gluonhq.com/products/javafx/